elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/timegrains/Time.java

```diff
@@ -24,7 +26,9 @@
 /**
  * Time date type for all analytic model time dimensions.
  */
-@Data
+@RequiredArgsConstructor
```

Comment:
Since we provide our own implementation of Hash, Equals and ToString using `@Data` annotation does not seem fit.